<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Program</class>
 <widget class="QDialog" name="Program">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="Master">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Master</string>
    </property>
    <widget class="QGroupBox" name="static_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>static_2</string>
     </property>
     <widget class="QGroupBox" name="default_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>default_2</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>90</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="power_layout">
        <item>
         <widget class="QPushButton" name="on_button">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Turn On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="off_button">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Turn Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="battery_box">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>311</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>battery_box</string>
       </property>
       <widget class="QProgressBar" name="battery_lvl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="battery_label">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>30</y>
          <width>47</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="clock">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>10</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="battery_msg">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>70</y>
         <width>221</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MV Boli</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Program Menu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="return_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>130</y>
         <width>158</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DENAS System</string>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>default_2</zorder>
    </widget>
    <widget class="QGroupBox" name="menu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>781</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>781</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="sub_menu">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QGridLayout" name="sub_grid">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="2" column="2">
          <widget class="QPushButton" name="b6">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nausea</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="b2">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Pain</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QPushButton" name="b8">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Joints</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="b3">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Throat</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="b1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Allergy</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="b5">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Cold</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="b7">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Trauma</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="b4">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Head</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="Line" name="line_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="Line" name="line_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="report">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>771</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="menu2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>781</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>781</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="sub_menu_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QGridLayout" name="sub_grid_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QProgressBar" name="progress">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Testing area for 5 seconds at 10Hz</string>
      </property>
     </widget>
    </widget>
    <zorder>menu</zorder>
    <zorder>static_2</zorder>
    <zorder>menu2</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
